### ISSUE TITLE
error when using reports, dashboar with filter period

### ISSUE THREAD

**Issue by mckaygerhard** [osTicket/osTicket#1347]
in osTicket i go to scp/dashboard.php  to see the reports, and i choose a previous day to filter

1) the date that appears in combobox are set to mm/dd/a yes an "a" its on the last...
2) then i see this error in apache server doc:
´´´
PHP Fatal error:  Call to a member function format() on a non-object in /opt/simpleticket/simpleticket-1.9/include/ajax.reports.php on line 177
´´´
i'm running lasted osticket from git, (also try with lasted release 1.9.3 and still happening)
i have enought hardware and php 5.3.12 with fascgi scaling, no suhoshin enable, no firewall in dedicated server

the tests where made in that browsers: icecat 7, chrome 30, iceweasel 22, firefox 29, firefox 17, firefox 7, and chromium 13 and 16, same error in all ....
´´´
Error: extStatus is not a function
Source File: chrome://priv/content/priv.js
Line: 292
´´´
in crhome:
GET http://170.10.1.21/simpleticket/scp/ajax.php/report/overview/graph?start=09%2F29%2Fa&period=now 500 (Internal Server Error)

**Comment by mckaygerhard** [comment]
hello, i migrate to 1.9.5 and this error now goes back.. i investigate and found the real problem

1) if installed and gos usage directly , no problem... 
2) if installed and goes to date format settings, a default format are in the fields correctly

**BUT** if i used diferent languaje, (i used es_ES) this happened:
1) if installed , then before usage, put spanish phar files and used without goind to date settings, all be fine
2) if installed , then before usage, put spanish phar files (or other) and goes to date format settings, a wrong default format are in fields

so then in conclusion, if users have different languaje in osticket and configure the date format without take care of that format, a worng format will be in fields by default for users, and if user are not a programer or admin avanced will not note that problem.

**Comment by greezybacon** [comment]
So the date format is wrong in the es_ES language pack?

**Comment by mckaygerhard** [comment]
i dont know .. there's the format include in the i18n file of each languaje?

**Comment by mckaygerhard** [comment]
i was thinking the format of the date and time was in the locale settings of the app , and not in the files of internationalization

**Comment by Chefkeks** [comment]
The formats are in the language packs since e.g. in Germany we use different date and time formats compared to others who use the US date and time format for example.

**Comment by mckaygerhard** [comment]
ok then i revised tomorrow the format of my lang packs, i have russian, greek and espanish... and post later here..

**Comment by mckaygerhard** [comment]
confirmed, in the file config.yaml there the format: d/m/a that's invalid! many translation files are wrong!

this issue are criticall and must be corrected hurry

**Comment by mckaygerhard** [comment]
also noted that some strings in the russian, greek, spanish are wrong, specially the date and time formats.. critical issue!

**Comment by mckaygerhard** [comment]
ping...

still this critical issue are unconfirmed!!!! the russian, spanish and greek lang packs are confirmed that have many wrong strings and formats.

i can help to solve that but i dont find here in github wiki rules to doit, also the lables are still "unconfirmed"

**Comment by ntozier** [comment]
http://i18n.osticket.com/

**Comment by mckaygerhard** [comment]
i try to find, but only find one string related, and its not the right to update... that cumbersome web interface does not allow me to find (actually not let me whatsoever) the string, it's not possible using thats web tool, in conclusion it is impossible to arrange those invalid strings without wasting time ..

confirmed invalid string langs are in russian, greek and spanihs languajes, make unnavailable the reports if users updates config page section, event not change the date, the input field will update to the invalid format always...

ping... label still unconfirmed, no milestone assigned

**Comment by mckaygerhard** [comment]
this errors still happened with all branches including the develop-next .. the label are still market as "unconfirmed"

**Comment by mckaygerhard** [comment]
i found another problems in languajes: #1832

**Comment by mckaygerhard** [comment]
ping? still maks as unconfirmed this issue and some reports are now in the line of comments

**Comment by greezybacon** [comment]
@mckaygerhard can you post your date format so I can try and trigger the issue?

**Comment by mckaygerhard** [comment]
why i must post my format, the error are in the i18n yaml files not on my corrected now files... !!! my new date format that fixed are: 

```
Time format:  "h:i a"  	  *  08:00 pm
Date format:  "Y-m-d"	  *  2015-03-30
Date and time format: "Y-m-d H:i"	  *  2015-03-30 20:00
Complete date and time format:  "D, j M Y. (d/m/y) h:ia"	  *  Mon, 30 Mar 2015. (30/03/15) 08:00pm
```

in the spanish template and greek i found "m/d/a", also "m/d/a g:i a" and both are incorrect, in the russian i have a "j" in the complete format but when i use got also and error..

**Comment by mckaygerhard** [comment]
this bug are still happened in 1.9.12, the date format of spanish translation are wrong!

**Comment by mckaygerhard** [comment]
another bug related with this are: #2531
the languaje files still have errors and the web tool to translate does not help in any due does not permit edit the strings that are bad

**Comment by mckaygerhard** [comment]
this issue still happened in 1.9.13, will happened for ever?

**Comment by ntozier** [comment]
1.9.13 was never formally released.  The dashboard has not been updated and has been on the to do list for a really long time.  In fact I recommend people not use the dashboard fairly regularly.

**Comment by mckaygerhard** [comment]
that its not the point, a year was passed and in both (1.9.X and 1.10.X) the problem persist!

**Comment by ntozier** [comment]
My point is that **No one has commit-ed to any sort of time frame to fix this issue**, nor will anyone commit to fixing the dashboard since there are better alternatives and much more higher priority items to address.

**Comment by mckaygerhard** [comment]
ok, so then i'll made a pull request for 1.9 brand due i'm not developer and i'm only have this release to work.. i know where are the paths of urls to fixed.. 

i hope some thing will commit my pull.. gimme a while and in few weeks i'll made the pull !

**Comment by mckaygerhard** [comment]
any progress? super important bug, but "other are more critical"? there are a more that 4000 issues seem here anre need some organization!

this bug are super-confirmed and the i18n template seems still are in problems.. i do not see the git for i18n collaboration.. puff

**Comment by mckaygerhard** [comment]
another bug closed @JediKev  and still a problem.. i18n files are still problematic.. as i reported in the history of this issue

**Comment by JediKev** [comment]
@mckaygerhard 

As stated before, this was closed due to the inactivity and lack of updates. If this is still an issue please re-open.

Cheers.
